### ISSUE TITLE
Block placements get stopped when map generates

### ISSUE THREAD

**Issue by osu** [Imanity-Software/ImanitySpigot3-git#113]
### Stack trace

Here is the error outputing on our plugin, we are also thinking that it might come from AsyncLight new addon 

https://i.imgur.com/NV0uUBO.png

### Plugin list

Custom mades one by us.

### Steps/models to reproduce (if known)

Here is the code used for the block generation when map is loaded

https://i.imgur.com/d42stBZ.png
https://i.imgur.com/q1Gyiqh.png

### ImanitySpigot3 version

2022.05.1 LTS BUILD 2

### Agreements

- [X] I am running the latest version of ImanitySpigot3 available from
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have compared the issue inbetween TacoSpigot 1.8.8 and ImanitySpigot3 And only ImanitySpigot3 occured this issue.

### Other

So basically we are trying to set a "spawn" for our players (UHC gamemode) in X: 0 Z:0 coords (invisible blocks) but while generating the blocks it just stops and outputs this error, we think that it could be linked to the spigot.
